src/org/labkey/test/tests/list/ListTest.java

```diff
@@ -751,6 +751,29 @@ public void testRemoveColumnOverAPI() throws Exception
                                 a.getRangeURI().endsWith(FieldDefinition.ColumnType.String.getConceptURI())));
     }
 
+    @Test
+    public void testChangeListName() throws Exception
+    {
+        List<FieldDefinition> cols = Arrays.asList(
+                new FieldDefinition("name", FieldDefinition.ColumnType.String),
+                new FieldDefinition("title", FieldDefinition.ColumnType.String),
+                new FieldDefinition("dewey", FieldDefinition.ColumnType.Decimal)
+        );
+        String listName = "remoteAPIBeforeRename";
+        FieldDefinition.LookupInfo info = new FieldDefinition.LookupInfo(getProjectName(), "lists", listName);
+        TestDataGenerator dgen = new TestDataGenerator(info)
+                .withColumns(cols);
+        DomainResponse createResponse = dgen.createList(createDefaultConnection(true), "key");
+        Domain listDomain = createResponse.getDomain();
+        listDomain.setName("remoteAPIAfterRename");
+
+        SaveDomainCommand saveCmd = new SaveDomainCommand(listDomain.getDomainId());
+        saveCmd.setDomainDesign(listDomain);
+        DomainResponse saveResponse = saveCmd.execute(createDefaultConnection(true), info.getFolder());
+
+        assertEquals("remoteAPIAfterRename", saveResponse.getDomain().getName());
```

Comment:
The test expects the rename to take here- you can rename a list over the UI- but the save domain doesn't seem to save it

Reply:
I think you need to set the name in the `options` object of the command. For some reason, it is only exposed when creating the domain (`CreateDomainCommand.setOptions()`). Might need to `@Ignore` this test until the API is expanded.

Reply:
ignoring for now